### PR TITLE
fix: Fix 404 broken link in doc.

### PR DESCRIPTION
**Pull request by shahzeb1**
## Describe this PR

Fix 404 broken link which can be found on this [page](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse?query=early%20access&page=1).

## Changes

Fix link to what I think it was meant to be.

### CODE REVIEW

File: content/800-data-platform/200-pulse/100-what-is-pulse.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ toc: true
 
 <Admonition type="warning">
 
-Pulse is currently in [Early Access](/data-platform/classic-projects/about/releases#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
+Pulse is currently in [Early Access](/docs/data-platform/platform-console/maturity-levels#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
```
Comment (ankur-arch) [inline content/800-data-platform/200-pulse/100-what-is-pulse.mdx]:
```suggestion
Pulse is currently in [Early Access](/data-platform/platform-console/maturity-levels#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
```

Comment (shahzeb1) [reply content/800-data-platform/200-pulse/100-what-is-pulse.mdx]: Is there something I need to do to approve the change? 

I'm not sure why the CI is failing on the PR.

Comment (ankur-arch) [reply content/800-data-platform/200-pulse/100-what-is-pulse.mdx]: Hey there @shahzeb1 👋 , I'm looking into it.

**Comment by jharrell** [comment]
@shahzeb1 thank you for your original PR and apologies for not merging it. We've since updated our docs site and this link is most likely fixed. If there are still issues, please don't hesitate to re-open.

**Pull request closed without merging**